ff_derive/src/lib.rs

```diff
@@ -1282,7 +1282,7 @@ fn prime_field_impl(
                     // `0xfff... >> REPR_SHAVE_BITS` overflows. So use `checked_shr` instead.
                     // This is always sufficient because we will have at most one spare limb
                     // to accommodate values of up to twice the modulus.
-                    tmp.0.as_mut()[#top_limb_index] &= 0xffffffffffffffffu64.checked_shr(REPR_SHAVE_BITS).unwrap_or(0);
+                    tmp.0[#top_limb_index] &= 0xffffffffffffffffu64.checked_shr(REPR_SHAVE_BITS).unwrap_or(0);
```

Comment:
Hmm, I do not understand what is going on here. But the `.as_mut()` was added in 5a48059a14ec82371c61935869b001f1200936c2 in mid-2018, so it was necessary for 2015 edition Rust but likely is no longer required even with an MSRV of 1.56 as our current release has. So I'll bring this in as a patch fix to 0.13.